App/VSTS/SettingsComponents/AccountDropdown.tsx

```diff
@@ -0,0 +1,61 @@
+/// <reference path="../../../office.d.ts" />
+import * as React from 'react';
+import { Provider, connect } from 'react-redux';
+import {updateSettings, updateSettingsLists, ISettingsInfo} from '../../Redux/LoginActions';
+import {Rest, Project } from '../../RestHelpers/rest';
+
+// other import statements don't work properly
+require('react-select/dist/react-select.css');
+let Select: any = require('react-select');
+
+interface ICombo {
+  dispatch?: any;
+  id?: string;
+  email?: string;
+  account?: string;
+  accounts?: ISettingsInfo[];
```

Comment:
account and accounts can be confusing. please call one account and other accountList